index.html

```diff
@@ -617,10 +617,10 @@ <h2 class="section-title" style="text-shadow: 1px 1px black;">Contact Us</h2>
             <ul class="contact-info-list list-inline">
                 <li class="item">
                     <span class="custom-icon icon-telephone_signal"></span>
-                    <span class="info"><a href="#" style="text-shadow: 1px 1px black;">(208) 858-3040</a></span>
+                    <span class="info"><a href="#" style="text-shadow: 1px 1px black;" id="phone-number">(208) 858-3040</a></span>
```

Comment:
@wingchunkf For future reference - try not to use `id` attributes on elements, I understand it's easy to create a selector for an element that has an ID but it's best practice to use a more specific selector, ie: `document.querySelector('span.info > a')`